test/verifiedTests/analysis/testGeneMCS/testGeneMCS.m

```diff
@@ -17,7 +17,7 @@
 cd(testDir);
 
 % define the solver packages to be used to run this test
-solverPkgs = {'ibm_cplex','gurobi'}; 
+solverPkgs = {'ibm_cplex'}; 
```

Comment:
May you actually still test `gurobi` and catch the error as explained [here](https://github.com/opencobra/cobratoolbox/blob/master/docs/source/guides/testGuide.md#test-if-a-function-throws-an-error-or-warning-message)?